### PR TITLE
$GOPATH and Path variable setup command is added

### PR DESCRIPTION
**Pull request by agnelvishal**
The default Go installation doesn't require to set the value for $GOPATH and doesn't add $GOPATH/bin to Path variable.  But it is needed to run `pow`, So added in the README.md

Link to Go Install where setting of Path variables and $GOPATH is not mentioned - https://golang.org/doc/install
Link to Go docs where setting of Path variables and $GOPATH is  mentioned -  https://golang.org/doc/gopath_code.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,12 @@ To build and install the CLI, run:
 ```bash
 $ make install-pow
 ```
-The binary will be placed automatically in `$GOPATH/bin` which in general is in `$PATH`, so you can immediately run `pow` in your terminal.
+The binary will be placed automatically in `$GOPATH/bin`You may have to set the Path variables using the below coomands
```
Comment (jsign) [inline README.md:73]: I think a `.` is missing in the middle of the line.

**Comment by jsign** [comment]
Do you mind signing DCO to consider this PR mergable?

**Comment by jsign** [comment]
Thanks @agnelvishal , the DCO is the only thing missing for merging :)

**Comment by agnelvishal** [comment]
@jsign I am unable to sign a commit which has already been committed. Should I close this and create a new pull request with the sign-off commit?

**Comment by jsign** [comment]
> @jsign I am unable to sign a commit which has already been committed. Should I close this and create a new pull request with the sign-off commit?

Did you execute these steps? :)
image

**Comment by jsign** [comment]
`git rebase HEAD~4 --signoff`
`git push --force-with-lease origin patch-1`

Notice the `~4`, it was `~3` some minutes ago.

**Comment by agnelvishal** [comment]
`git rebase HEAD~4 --signoff` gives me `error: unknown option 'signoff'`
So, I tried `git rebase HEAD~4 -S` 
and then `git push --force-with-lease origin patch-1` gives `ERROR: Permission to textileio/powergate.git denied to agnelvishal.
`

`

**Comment by jsign** [comment]
Maybe you can try with another PR. Recall to `git commit -S -s -m 'commit message'` for the commit.

**Comment by agnelvishal** [comment]
@jsign Created new pull request in https://github.com/textileio/powergate/pull/707 . Closing this one

**Pull request closed without merging**